r3almX_backend\realtime_service\DigestionBroker.py:20: error: Incompatible types in assignment (expression has type "None", variable has type "AsyncSession")  [assignment]
r3almx_backend\realtime_service\DigestionBroker.py:20: error: Incompatible types in assignment (expression has type "None", variable has type "AsyncSession")  [assignment]
r3almX_backend\auth_service\user_handler_utils.py:18: error: Argument 1 to "filter" of "Select" has incompatible type "bool | ColumnElement[bool]"; expected "ColumnElement[bool] | _HasClauseElement[bool] | SQLCoreOperations[bool] | ExpressionElementRole[bool] | TypedColumnsClauseRole[bool] | Callable[[], ColumnElement[bool]] | LambdaElement"  [arg-type]
r3almX_backend\auth_service\user_handler_utils.py:23: error: Argument 1 to "filter" of "Select" has incompatible type "bool | ColumnElement[bool]"; expected "ColumnElement[bool] | _HasClauseElement[bool] | SQLCoreOperations[bool] | ExpressionElementRole[bool] | TypedColumnsClauseRole[bool] | Callable[[], ColumnElement[bool]] | LambdaElement"  [arg-type]
r3almX_backend\auth_service\user_handler_utils.py:24: error: Incompatible return value type (got "User | None", expected "User")  [return-value]
r3almX_backend\auth_service\user_handler_utils.py:28: error: Argument 1 to "filter" of "Select" has incompatible type "bool | ColumnElement[bool]"; expected "ColumnElement[bool] | _HasClauseElement[bool] | SQLCoreOperations[bool] | ExpressionElementRole[bool] | TypedColumnsClauseRole[bool] | Callable[[], ColumnElement[bool]] | LambdaElement"  [arg-type]
r3almX_backend\auth_service\user_handler_utils.py:29: error: Incompatible return value type (got "User | None", expected "User")  [return-value]
r3almx_backend\auth_service\user_handler_utils.py:18: error: Argument 1 to "filter" of "Select" has incompatible type "bool | ColumnElement[bool]"; expected "ColumnElement[bool] | _HasClauseElement[bool] | SQLCoreOperations[bool] | ExpressionElementRole[bool] | TypedColumnsClauseRole[bool] | Callable[[], ColumnElement[bool]] | LambdaElement"  [arg-type]
r3almx_backend\auth_service\user_handler_utils.py:23: error: Argument 1 to "filter" of "Select" has incompatible type "bool | ColumnElement[bool]"; expected "ColumnElement[bool] | _HasClauseElement[bool] | SQLCoreOperations[bool] | ExpressionElementRole[bool] | TypedColumnsClauseRole[bool] | Callable[[], ColumnElement[bool]] | LambdaElement"  [arg-type]
r3almx_backend\auth_service\user_handler_utils.py:24: error: Incompatible return value type (got "User | None", expected "User")  [return-value]
r3almx_backend\auth_service\user_handler_utils.py:28: error: Argument 1 to "filter" of "Select" has incompatible type "bool | ColumnElement[bool]"; expected "ColumnElement[bool] | _HasClauseElement[bool] | SQLCoreOperations[bool] | ExpressionElementRole[bool] | TypedColumnsClauseRole[bool] | Callable[[], ColumnElement[bool]] | LambdaElement"  [arg-type]
r3almx_backend\auth_service\user_handler_utils.py:29: error: Incompatible return value type (got "User | None", expected "User")  [return-value]
r3almX_backend\auth_service\auth_utils.py:86: error: Incompatible types in assignment (expression has type "Any | None", variable has type "str")  [assignment]
r3almX_backend\auth_service\auth_utils.py:92: error: Argument "email" to "get_user_by_email" has incompatible type "str | None"; expected "str"  [arg-type]
r3almx_backend\auth_service\auth_utils.py:86: error: Incompatible types in assignment (expression has type "Any | None", variable has type "str")  [assignment]
r3almx_backend\auth_service\auth_utils.py:92: error: Argument "email" to "get_user_by_email" has incompatible type "str | None"; expected "str"  [arg-type]
r3almX_backend\auth_service\auth_routes.py:29: error: Cannot determine type of "auth_router"  [has-type]
r3almX_backend\auth_service\auth_routes.py:83: error: Cannot determine type of "auth_router"  [has-type]
r3almX_backend\auth_service\auth_routes.py:100: error: Argument "user_id" to "create_auth_data" has incompatible type "str | Column[UUID]"; expected "str"  [arg-type]
r3almX_backend\auth_service\auth_routes.py:107: error: Cannot determine type of "auth_router"  [has-type]
r3almX_backend\auth_service\auth_routes.py:116: error: Incompatible types in assignment (expression has type "Any | None", variable has type "str")  [assignment]
r3almX_backend\auth_service\auth_routes.py:132: error: Cannot determine type of "auth_router"  [has-type]
r3almX_backend\auth_service\auth_routes.py:139: error: Argument 2 to "get_user_by_email" has incompatible type "Any | None"; expected "str"  [arg-type]
r3almX_backend\auth_service\auth_routes.py:156: error: Cannot determine type of "auth_router"  [has-type]
r3almX_backend\auth_service\auth_routes.py:178: error: Cannot determine type of "auth_router"  [has-type]
r3almX_backend\auth_service\auth_routes.py:185: error: Argument 2 to "get_user_by_email" has incompatible type "Any | None"; expected "str"  [arg-type]
r3almX_backend\auth_service\auth_routes.py:198: error: Cannot determine type of "auth_router"  [has-type]
r3almX_backend\auth_service\auth_routes.py:236: error: Cannot determine type of "auth_router"  [has-type]
r3almX_backend\chat_service\room_service\room_management_endpoints.py:24: error: Cannot determine type of "rooms_service"  [has-type]
r3almX_backend\chat_service\room_service\room_management_endpoints.py:42: error: Incompatible types in assignment (expression has type "list[str] | ColumnElement[NoReturn]", variable has type "list[str] | Column[NoReturn]")  [assignment]
r3almX_backend\chat_service\room_service\room_management_endpoints.py:48: error: Cannot determine type of "rooms_service"  [has-type]
r3almX_backend\chat_service\room_service\room_management_endpoints.py:54: error: Cannot determine type of "rooms_service"  [has-type]
r3almX_backend\chat_service\room_service\room_management_endpoints.py:60: error: Cannot determine type of "rooms_service"  [has-type]
r3almX_backend\chat_service\room_service\room_management_endpoints.py:77: error: Cannot determine type of "rooms_service"  [has-type]
r3almX_backend\chat_service\room_service\room_management_endpoints.py:98: error: Cannot determine type of "rooms_service"  [has-type]
r3almX_backend\chat_service\invite_system\invite_system_endpoint.py:18: error: Cannot determine type of "invite_system"  [has-type]
r3almX_backend\chat_service\invite_system\invite_system_endpoint.py:50: error: Cannot determine type of "invite_system"  [has-type]
r3almX_backend\chat_service\invite_system\invite_system_endpoint.py:95: error: Cannot determine type of "invite_system"  [has-type]
r3almX_backend\chat_service\channel_system\channel_management_endpoints.py:33: error: Cannot determine type of "channel_manager"  [has-type]
r3almX_backend\chat_service\channel_system\channel_management_endpoints.py:55: error: Cannot determine type of "channel_manager"  [has-type]
r3almX_backend\chat_service\channel_system\channel_management_endpoints.py:70: error: Name "create_channel" already defined on line 33  [no-redef]
r3almX_backend\chat_service\channel_system\channel_management_endpoints.py:70: error: Cannot determine type of "channel_manager"  [has-type]
r3almX_backend\chat_service\channel_system\channel_management_endpoints.py:79: error: Value of type "Coroutine[Any, Any, Any]" must be used  [unused-coroutine]
r3almX_backend\chat_service\channel_system\channel_management_endpoints.py:79: note: Are you missing an await?
r3almX_backend\chat_service\channel_system\channel_management_endpoints.py:92: error: Cannot determine type of "channel_manager"  [has-type]
r3almX_backend\chat_service\channel_system\channel_management_endpoints.py:138: error: Name "delete_channel" already defined on line 98  [no-redef]
r3almx_backend\auth_service\auth_routes.py:22: error: Incompatible import of "UsersConfig" (imported name has type "type[r3almx_backend.auth_service.Config.UsersConfig]", local name has type "type[r3almX_backend.auth_service.Config.UsersConfig]")  [assignment]
r3almx_backend\auth_service\auth_routes.py:100: error: Argument "user_id" to "create_auth_data" has incompatible type "str | Column[UUID]"; expected "str"  [arg-type]
r3almx_backend\auth_service\auth_routes.py:116: error: Incompatible types in assignment (expression has type "Any | None", variable has type "str")  [assignment]
r3almx_backend\auth_service\auth_routes.py:139: error: Argument 2 to "get_user_by_email" has incompatible type "Any | None"; expected "str"  [arg-type]
r3almx_backend\auth_service\auth_routes.py:185: error: Argument 2 to "get_user_by_email" has incompatible type "Any | None"; expected "str"  [arg-type]
r3almx_backend\chat_service\room_service\room_management_endpoints.py:42: error: Incompatible types in assignment (expression has type "list[str] | ColumnElement[NoReturn]", variable has type "list[str] | Column[NoReturn]")  [assignment]
r3almx_backend\chat_service\channel_system\channel_management_endpoints.py:70: error: Name "create_channel" already defined on line 33  [no-redef]
r3almx_backend\chat_service\channel_system\channel_management_endpoints.py:79: error: Value of type "Coroutine[Any, Any, Any]" must be used  [unused-coroutine]
r3almx_backend\chat_service\channel_system\channel_management_endpoints.py:79: note: Are you missing an await?
r3almx_backend\chat_service\channel_system\channel_management_endpoints.py:138: error: Name "delete_channel" already defined on line 98  [no-redef]
r3almX_backend\realtime_service\chat_service.py:56: error: Incompatible types in assignment (expression has type "Any | None", variable has type "str")  [assignment]
r3almX_backend\realtime_service\chat_service.py:60: error: Incompatible return value type (got "JWTError", expected "User")  [return-value]
r3almX_backend\realtime_service\chat_service.py:226: error: "Coroutine[Any, Any, Any]" has no attribute "username"  [attr-defined]
r3almX_backend\realtime_service\chat_service.py:235: error: Cannot determine type of "realtime"  [has-type]
r3almX_backend\realtime_service\chat_service.py:250: error: Cannot determine type of "realtime"  [has-type]
r3almX_backend\realtime_service\chat_service.py:266: error: Incompatible default for argument "length" (default has type "int", argument has type Never)  [assignment]
r3almX_backend\realtime_service\chat_service.py:275: error: Argument 3 to "add_message_to_queue" of "RoomManager" has incompatible type "str | Column[UUID]"; expected "str"  [arg-type]
r3almX_backend\realtime_service\connection_service.py:21: error: Incompatible types in assignment (expression has type "Any | None", variable has type "str")  [assignment]
r3almX_backend\realtime_service\connection_service.py:25: error: Incompatible return value type (got "JWTError", expected "User")  [return-value]
r3almX_backend\realtime_service\connection_service.py:93: error: "Connection" has no attribute "connection_cache_list"  [attr-defined]
r3almX_backend\realtime_service\connection_service.py:111: error: Cannot determine type of "realtime"  [has-type]
r3almX_backend\realtime_service\connection_service.py:118: error: Cannot determine type of "realtime"  [has-type]
r3almX_backend\realtime_service\connection_service.py:124: error: Invalid index type "str | Column[UUID]" for "dict[str, WebSocket]"; expected type "str"  [index]
r3almX_backend\realtime_service\main.py:6: error: Name "redis" already defined (by an import)  [no-redef]
r3almX_backend\inst_modifiers.py:8: error: Cannot determine type of "r3almX"  [has-type]
r3almX_backend\inst_modifiers.py:16: error: Name "logger" is not defined  [name-defined]
r3almX_backend\search_service\search_endpoints.py:38: error: Name "get_friends" already defined on line 11  [no-redef]
r3almX_backend\friends_service\friends_endpoint.py:34: error: Item "ColumnElement[Any]" of "list[str] | ColumnElement[Any]" has no attribute "__iter__" (not iterable)  [union-attr]
r3almX_backend\friends_service\friends_endpoint.py:61: error: No overload variant of "__and__" of "bool" matches argument type "ColumnElement[bool]"  [operator]
r3almX_backend\friends_service\friends_endpoint.py:61: note: Possible overload variants:
r3almX_backend\friends_service\friends_endpoint.py:61: note:     def __and__(self, bool, /) -> bool
r3almX_backend\friends_service\friends_endpoint.py:61: note:     def __and__(self, int, /) -> int
r3almX_backend\friends_service\friends_endpoint.py:61: note: Both left and right operands are unions
r3almX_backend\friends_service\friends_endpoint.py:61: error: Item "list[str]" of "list[str] | Column[NoReturn]" has no attribute "contains"  [union-attr]
r3almx_backend\realtime_service\connection_service.py:21: error: Incompatible types in assignment (expression has type "Any | None", variable has type "str")  [assignment]
r3almx_backend\realtime_service\connection_service.py:25: error: Incompatible return value type (got "JWTError", expected "User")  [return-value]
r3almx_backend\realtime_service\connection_service.py:93: error: "Connection" has no attribute "connection_cache_list"  [attr-defined]
r3almx_backend\realtime_service\connection_service.py:124: error: Invalid index type "str | Column[UUID]" for "dict[str, WebSocket]"; expected type "str"  [index]
r3almx_backend\realtime_service\chat_service.py:56: error: Incompatible types in assignment (expression has type "Any | None", variable has type "str")  [assignment]
r3almx_backend\realtime_service\chat_service.py:60: error: Incompatible return value type (got "JWTError", expected "User")  [return-value]
r3almx_backend\realtime_service\chat_service.py:226: error: "Coroutine[Any, Any, Any]" has no attribute "username"  [attr-defined]
r3almx_backend\realtime_service\chat_service.py:266: error: Incompatible default for argument "length" (default has type "int", argument has type Never)  [assignment]
r3almx_backend\realtime_service\chat_service.py:275: error: Argument 3 to "add_message_to_queue" of "RoomManager" has incompatible type "str | Column[UUID]"; expected "str"  [arg-type]
r3almx_backend\proj_logger.py:9: error: Name "logger" already defined (by an import)  [no-redef]
r3almx_backend\proj_logger.py:23: error: Incompatible types in assignment (expression has type "Logger", variable has type Module)  [assignment]
r3almx_backend\proj_logger.py:24: error: Module has no attribute "setLevel"  [attr-defined]
r3almx_backend\proj_logger.py:35: error: Item "None" of "Address | None" has no attribute "host"  [union-attr]
r3almx_backend\inst_modifiers.py:16: error: Name "logger" is not defined  [name-defined]
r3almx_backend\search_service\search_endpoints.py:38: error: Name "get_friends" already defined on line 11  [no-redef]
r3almx_backend\realtime_service\main.py:6: error: Name "redis" already defined (by an import)  [no-redef]
r3almx_backend\realtime_service\main.py:6: error: Incompatible import of "NotificationSystem" (imported name has type "type[r3almx_backend.realtime_service.connection_service.NotificationSystem]", local name has type "type[r3almX_backend.realtime_service.connection_service.NotificationSystem]")  [assignment]
r3almx_backend\realtime_service\main.py:6: error: Incompatible import of "notification_system" (imported name has type "r3almx_backend.realtime_service.connection_service.NotificationSystem", local name has type "r3almX_backend.realtime_service.connection_service.NotificationSystem")  [assignment]
r3almx_backend\friends_service\friends_endpoint.py:34: error: Item "ColumnElement[Any]" of "list[str] | ColumnElement[Any]" has no attribute "__iter__" (not iterable)  [union-attr]
r3almx_backend\friends_service\friends_endpoint.py:61: error: No overload variant of "__and__" of "bool" matches argument type "ColumnElement[bool]"  [operator]
r3almx_backend\friends_service\friends_endpoint.py:61: note: Possible overload variants:
r3almx_backend\friends_service\friends_endpoint.py:61: note:     def __and__(self, bool, /) -> bool
r3almx_backend\friends_service\friends_endpoint.py:61: note:     def __and__(self, int, /) -> int
r3almx_backend\friends_service\friends_endpoint.py:61: note: Both left and right operands are unions
r3almx_backend\friends_service\friends_endpoint.py:61: error: Item "list[str]" of "list[str] | Column[NoReturn]" has no attribute "contains"  [union-attr]
Found 95 errors in 26 files (checked 46 source files)
